schema/rss-sled-plan.json

```diff
@@ -1043,8 +1043,11 @@
       "properties": {
         "id": {
           "description": "Uuid of the Sled Agent to be created.",
-          "type": "string",
-          "format": "uuid"
+          "allOf": [
+            {
+              "$ref": "#/definitions/TypedUuidForSledKind"
+            }
+          ]
```

Comment:
This and the following schema are compatible changes, because newtype UUIDs serialize the same way as regular ones.